### PR TITLE
Bug 507724: Support HiDPI icons

### PR DESCRIPTION
**Pull request by BeckerWdf**
- Copy SVG to images repo
- Create missing SVGs

Signed-off-by: Matthias Becker <ma.becker@sap.com>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=507724

**Comment by Godin** [comment]
@BeckerWdf Thank you for your contribution! 👍  

While I'm fixing build of master that also a cause of build failure for this PR, and while checking new icons - a little organizational question: being a newbie in Eclipse Foundation, I'm wondering why move SVG sources away from project repository? Is it a common practice? Maybe there is some reference where I can read more about this?

**Comment by BeckerWdf** [comment]
All the other Eclipse projects do store their SVGs in the central images repo: https://git.eclipse.org/r/#/admin/projects/platform/eclipse.platform.images
This has the advantage, that all Eclipse SVGs are in one location. So when you search for an existing icon you only have to search one repo.
This repo also contains 
- the Maven Mojo for rendering PNGs out of the SVGs
- documentation in the README.md files
I don't know of any other reference about this topic.

**Comment by BeckerWdf** [comment]
- coverage*-icons added
- fixed typo  read -> red
- If a gif is referred to by a view or editor, we should not delete it, as the e4 application model caches the image reference and will use the old gif unless you specify the "cleanPersistedState" flag or use a new workspace.
- Regarding the testng_launch.gif:  I did not know what to do with this icon. The style looks quite different to all the other icons. First I thought this is the "official" icon of "Test NG" but it seems that there is no official "Test NG" icon. So I simply skipped it.

File: org.eclipse.eclemma.ui/src/org/eclipse/eclemma/internal/ui/EclEmmaUIPlugin.java [hunk 1]
```diff
@@ -57,21 +57,21 @@
...
-      "icons/full/ovr16/coverage06.gif", //$NON-NLS-1$
-      "icons/full/ovr16/coverage07.gif" //$NON-NLS-1$
+      "icons/full/ovr16/coverage00.png", //$NON-NLS-1$
+      "icons/full/ovr16/coverage01.png", //$NON-NLS-1$
+      "icons/full/ovr16/coverage02.png", //$NON-NLS-1$
+      "icons/full/ovr16/coverage03.png", //$NON-NLS-1$
+      "icons/full/ovr16/coverage04.png", //$NON-NLS-1$
+      "icons/full/ovr16/coverage05.png", //$NON-NLS-1$
+      "icons/full/ovr16/coverage06.png", //$NON-NLS-1$
+      "icons/full/ovr16/coverage07.png" //$NON-NLS-1$
```
Comment (BeckerWdf) [inline org.eclipse.eclemma.ui/src/org/eclipse/eclemma/internal/ui/EclEmmaUIPlugin.java]: Fixed

**Comment by Godin** [comment]
> If a gif is referred to by a view or editor, we should not delete it, as the e4 application model caches the image reference and will use the old gif unless you specify the "cleanPersistedState" flag or use a new workspace.

Good to know. However this will be the first version released under Eclipse Foundation umbrella, so after change of IDs and so there not supposed to be any reference on old GIFs. And we already removed GIFs in fb0a457a8fbfb45ae96518d1ff9d01d8c67a7a26

**Comment by BeckerWdf** [comment]
Ok. Then let's delete them.

**Comment by BeckerWdf** [comment]
I also just found out that I copied the coverage* PNG into the wrong folder. I also fixed this with 576cfe

**Comment by Godin** [comment]
There still was a typo - "rebar" instead of "redbar". Fixed it.

Also as we don't have commit rights on eclipse.platform.images project, we'll keep SVGs here till someone with sufficient rights approves merge of https://git.eclipse.org/r/#/c/96176/

Thank you for your contribution.

**Comment by vogella** [comment]
FYI currently https://git.eclipse.org/r/#/admin/projects/platform/eclipse.platform.images is only used for sub-projects of the Eclipse project https://www.eclipse.org/eclipse/ IMHO it is better to store the images in the correct project

**Comment by Godin** [comment]
@vogella thank you for the information. So actually good that removal wasn't merged :wink:

**Pull request merged**